### ISSUE TITLE
Builds made by the dev build server appear broken. 

### ISSUE THREAD

**Issue by fritogotlayed** [DUWS-R-Team/DUWS-R#31]
Builds of the pbo by the dev build server appear broken. Figure out what's going on and fix it.

EXPECTED OUTCOME
Builds from the build server work.

**Comment by fritogotlayed** [comment]
![image](https://cloud.githubusercontent.com/assets/5406580/5932168/72720f2a-a66e-11e4-844f-a202626a741e.png)

I'm getting this when I use any build (tested with 0.0.0.9) from the [dev download](http://www.maddonkeysoftware.com/DUWS-R/Downloads.aspx) page and try to run a multiplayer game. It works fine single player..

**Comment by fritogotlayed** [comment]
I did a little bit of digging and this appears to have something to do with the process of the agent actually building the pbo. If I build the pbo manually from the source on the build server it works fine.

**Comment by fritogotlayed** [comment]
Removing this from a milestone since it is not directly related to a release.

**Comment by fritogotlayed** [comment]
I did a bit more digging into this and it looks like the current tool, [PBO Manager](http://www.armaholic.com/page.php?id=16369), might export to a different PBO format than Arma 3 does. This nuance might be what is causing "screwy" things to happen when it is built by different mechanisms.

**Comment by hatstand0** [comment]
I did [some testing](http://ci.dev.nihil-ad-rem.net/job/DUWS-R/) using [Mikero's tools](http://forums.bistudio.com/showthread.php?118821-Mikero-s-Dos-Tools) (And Wine), they seem to provide builds that work in Multiplayer just fine.

**Comment by fritogotlayed** [comment]
@hatstand0 was there any special configuring you had to do? I have `DoOgg.1.00.4.55`, `'DePbo.4.91.0.11` and `MakePbo.1.88.4.85` installed. On the machine the `Visual C++ Redist.` is installed as well. The build system is Win 2000 64bit. 

When I try to run MakePbo I get a dialog complaining about `MSVCR110.dll` not being installed... I think this is originally why I didn't go this route. Also, the above are the only Mikero tools installed on the box.

**Comment by fritogotlayed** [comment]
@hatstand0 Derp. I had the Visual C++ Redist. x64 installed. Not x86.

**Comment by fritogotlayed** [comment]
I just tried MakePbo w/ the same options as configured on the example provided by @hatstand0. I'm still getting the same error with local testing. Will continue to muck with this as time permits.

**Comment by hatstand0** [comment]
I should have posted the actual command I'm using, sorry: `MakePbo.exe -A -N -P source DUWS-R.altis.pbo`

**Comment by fritogotlayed** [comment]
@hatstand0 It's all good. Your testing link was helpful b/c I could see the commands that were run from the build log. I ran the same commands on the build machine from the command line and resultant pbo worked. I tied that into the build process and the resultant pbo did not work. :-( This thing has me stumped. 

If you're willing could you take your pbo and attempt to run a internet DUWS-R game w/ a password. I never actually get to the screen where you can select the placement for yoru HQ or let it be randomly placed.

**Comment by fritogotlayed** [comment]
![image](https://cloud.githubusercontent.com/assets/5406580/6054473/2cdfbeee-acb3-11e4-86c2-38f2fc3398c8.png)

@hatstand0 @bolbies @phr0gz @naitp 

Above I pasted a diff captured by Beyond Compare 4. I took all the code from DUWS-R and exported it to both single player and multi player missions. When I ran the export for single player the diff showed 100% of the 'code' the same. After I exported the multi-player version though I noticed the above is different. Do any of you know why this is different? Is this some undocumented feature from bohemia? 

My gut check is that the right side (the multiplayer version) contains some extra packages that are used in multiplayer. I may do some playing around locally on Friday to see if having this built into the PBO will affect single player adversely. If not, and no one has chimed in on here, I'll make a PR to add these into master. 

On a side note... I've also built a C# PBO builder that matches the file format exported by ArmA 3. I'll probably open it up on Git Hub soon as well since it seems like there's some demand out there for something that matches the A3 format.
